test/src/touchscreen.spec.ts

```diff
@@ -51,4 +53,61 @@ describe('Touchscreen', function () {
       })
     ).toEqual(['Touchstart: 0', 'Touchend: 0']);
   });
+
+  it('should report press', async () => {
+    const { page, server } = getTestState();
+    const iPhone = KnownDevices['iPhone 6']!;
+    await page.emulate(iPhone);
+    await page.goto(server.PREFIX + '/input/touches.html');
+    await page.touchscreen.press(0, 0, 5);
+    expect(
+      await page.evaluate(() => {
+        return (globalThis as any).getResult();
+      })
+    ).toEqual(['Touchstart: 0', 'Touchend: 0']);
```

Comment:
I don't see any indication in the test that it was actually a press and not  a tap? the expectation looks exactly as the previous test.

Reply:
I upload a new html for press